### PR TITLE
This is my finish Assignment 1 for Java

### PR DESCRIPTION
**Pull request by Rtihai**
Hi Derrick, it's my finish Assignment.

Thank you.

### CODE REVIEW

File: src/main/java/ca/ciccc/Assignment1.java [hunk 1]
```diff
@@ -25,8 +29,10 @@ public static String fahrenheitToCelsius(double degree) {
      * @return "1000.0 inches is 25.4 meters"
      */
     public static String inchesToMeters(int inches) {
+        double number = 1000;
```
Comment (amagin006) [inline src/main/java/ca/ciccc/Assignment1.java]: Take argument (int inches) instead of hard code.

File: src/main/java/ca/ciccc/Assignment1.java [hunk 2]
```diff
@@ -49,8 +63,12 @@ public static String addDigits(int number) {
      * @return "3456789 minutes is approximately 6 years and 210 days"
      */
     public static String minsToYearsDays(int mins) {
+        int timeScore = 3456789;
```
Comment (amagin006) [inline src/main/java/ca/ciccc/Assignment1.java]: You can use variable instead of hardcode number.

File: src/main/java/ca/ciccc/Assignment1.java [hunk 3]
```diff
@@ -127,7 +169,22 @@ public static String powers(int number) {
...
      */
     public static String arithmetic(int a, int b) {
-
-        return "";
+        int aa = 25;
+        int bb = 5;
+        int sum = aa + bb;
+        int diff = aa - bb;
+        int prod = aa * bb;
+        double aver = (aa + bb) / 2;
```
Comment (amagin006) [inline src/main/java/ca/ciccc/Assignment1.java]: You need to cast from int to double if you want correct double type answer.

File: src/main/java/ca/ciccc/Assignment1.java [hunk 4]
```diff
@@ -37,8 +43,16 @@ public static String inchesToMeters(int inches) {
      * @return "The sum of all digits in 565 is 16"
      */
     public static String addDigits(int number) {
+        int sampleInput = 565;
```
Comment (itzderr) [inline src/main/java/ca/ciccc/Assignment1.java]: `int sampleInput = number`
You need to use the function argument as input.

**Pull request closed without merging**